src/cli.c

```diff
@@ -15,18 +15,24 @@
 #endif
 
 #include "storj.h"
+#include "crypto.h"
 #include "cli_callback.c"
 
 //#define debug_enable
 
 #define STORJ_THREADPOOL_SIZE "64"
 
+#define STORJ_CLEANUP_SET_STATUS_IF_LAST_ERROR(value) \
+if (strcmp("", *STORJ_LAST_ERROR) != 0) { \
+    status = value; \
+    goto clean_variables; \
+}\
+
 typedef struct {
-    char *user;
-    char *pass;
     char *host;
-    char *mnemonic;
-    char *key;
+    char *apikey;
+    char *enc_access;
+    char *user_passphrase;
 } user_options_t;
 
 #ifndef errno
```

Comment:
The `HELP_TEXT` a few lines below need an update. At least, we should not have the `register` command anymore.

Reply:
`CLI_VERSION` should be also updated to something like "libstorj-2.0.0-v3-beta"